salt/transport/zeromq.py

```diff
@@ -392,6 +392,9 @@ async def consume(running):
                     if msg:
                         try:
                             await callback(msg)
+                        except TypeError as exc:  # pylint: disable=broad-except
+                            if callback is not None:
+                                log.error("Exception while running callback %s", exc, exc_info=True)
```

Comment:
```suggestion
                                log.exception("Exception while running callback: %s", callback)
```
Log the callback, and `log.exception` will make sure `exc_info` is also shown.